lib/routing.js

```diff
@@ -61,6 +61,11 @@ exports.routes = [
     method: 'POST',
     path: v('/verify'),
     config: require('./routes/verify')
+  },
+  {
+    method: 'GET',
+    path: v('/jwks'),
+    config: require('./routes/jwks')
```

Comment:
This `/jwks` route needs some tests, in particular:

* Testing that we don't send the private key information as part of this document
* Testing that the oldKey is correctly served when present